src/OpenFeature/Model/EvaluationContext.cs

```diff
@@ -11,26 +11,29 @@ namespace OpenFeature.Model
     /// <seealso href="https://github.com/open-feature/spec/blob/v0.5.2/specification/sections/03-evaluation-context.md">Evaluation context</seealso>
     public sealed class EvaluationContext
     {
+        /// <summary>
+        /// The index for the "targeting key" property when the EvaluationContext is serialized or expressed as a dictionary.
+        /// </summary>
+        public readonly static string TargetingKeyIndex = "targetingKey";

```

Comment:
```suggestion
        public const string TargetingKeyIndex = "targetingKey";

```

Can we make it `const` here for some performance gains? Do we want this to be exposed, or can it be `internal`?

Reply:
Yea i don't think we need to expose this publicly

Reply:
I was going back and forth on this, so I am fine to not expose it.

Reply:
Yup. Let's make `internal` :)

Reply:
Done